### PR TITLE
Updated website

### LINKED ISSUE #29: Are there working samples?
I have installed bootstrap-wysiwyg-steveathon from bower. Then I opened a file `examples/basic.html` and tried to write some chars into edit field. I've got a javascript exception:

    NotFoundError: Node was not found
    document.execCommand(command, 0, args);

browser: firefox 37.0.2

Are there any working samples how to use this component?

### PR DESCRIPTION
**Pull request by RandomlyKnighted**
This PR fixes #29, and #39 by adding a live demo to the website. In order to do this I made the Jekyll template Bootstrap 3 compatible. I migrated all dependencies over to CDN except for jQuery.hotkeys. Since jQuery.hotkeys does not have a CDN I added it using bower so that it does not have to be managed manually.

Finally, I have updated the website so that the basic usage now contains all the information on the Mindmup website. This way all the information is contained on one page and doesn't have to point to the Mindmup website other than to show that this repo was originally made by them.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -191,6 +275,70 @@
       </div>
     </div>
 
-  
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/js/bootstrap.min.js"></script>
+    <script src="https://cdn.rawgit.com/google/code-prettify/master/loader/run_prettify.js"></script>
+    <script src="bower_components/jquery.hotkeys/jquery.hotkeys.js"></script>
```
Comment (steveathon) [inline index.html]: We can probably use the CDN version of this too?

Comment (RandomlyKnighted) [inline index.html]: I looked for an offical CDN but didn't find one. Do you know of one that I don't?

Comment (RandomlyKnighted) [inline index.html]: @steveathon I'm in the process of submitting a PR to the [cdnjs.com repo](https://github.com/cdnjs/cdnjs) containing the jQuery.hotkeys library. As soon as I get it done and it's approved I'll update my code to use the CDN link and I'll remove Bower while I'm at it.

**Comment by steveathon** [comment]
This is good to go. I've merged it in. Thanks for the great work on this @RandomlyKnighted

**Pull request merged**